plane/source/docs/takeoff-landingpage.rst

```diff
@@ -0,0 +1,55 @@
+.. _takeoff-landingpage:
+
+=======
+Takeoff
+=======
+
+Taking the plane off can be accomplished in many ways: Rolling Takeoff, Hand Launch, Bungee, or VTOL (Verical Takeoff and Landing) if a :ref:`quadplane-support`. And it can be done manually or automatically.
+
+Rolling Takeoff
+===============
+
+In this case, the plane has landing gear with steerable nose gear, usually coupled to the rudder control and the plane is usually launched from a smooth grass or paved surface. Once aligned nose into the wind, throttle is gradually increased to takeoff value (usually full throttle) while the aircraft direction is controlled by rudder. Once sufficient airspeed to lift off is obtained, slight up elevator will rotate the nose to climb attitude.
+
+MANUAL or FBWA modes are most commonly used during takeoffs.
+
+For automatic takeoffs see :ref:`automatic-takeoff` section.
+
+Hand Launch
+===========
+
+Many planes do not have landing gear, or a smooth runway is not available. Hand launching is a very common takeoff method, but also can be very frustrating if not setup and executed properly.
+
+The keys to a successful hand launch are :
+
+- :ref:`Proper CG <guide-center-of-gravity>` 
+- Having flying surfaces centered and neutral at neutral RC stick positions
+- Having the "level" calibration setup such that the level horizon (0 deg pitch) in FBWA mode results in the wing chord line being a few degrees positive, which will result in a positive angle of attack in "level" flight, the normally cruising attitude for a plane.
+- Using FBWA mode or :ref:`automatic-takeoff` . While launching in MANUAL can be done, using a stabilized mode will eliminate a lot of pilot effort to overcome trim issues (see Tip at end of this page). If using FBWA mode, a few degrees of up elevator (~10%) during the launch will result in an automatic climb. Add this via a switch or stick pressure. Do not change TX trim positions used during RC calibration since it affects all other modes of flight.
+- Throttle up and firmly throw the plane level, not nose pointed up. Common mistakes are pointing the nose up, throwing it toward the ground, or rotating your wrist like a football throw. In the first second or so, the plane has low airspeed and the flying surfaces have no authority to correct non-level attitude until it accelerates to flying speed.
+
```

Comment:
Add another point about launching into the wind. Hand-launches with a tailwind are often disastrous because the airspeed is negative to start and requires a lot of "delta V" or velocity change to get the control surfaces to do anything.

Reply:
yep...should add nose into the wind...thanks

Reply:
Oh... Running takeoffs are another beef of mine related to tips.

I think we should recommend NOT doing them as the throw technique is compromised so much for so little gain in velocity. Instead, find a location with no obstacles in front of you and do a step forward like tossing (or throwing) a ball. Just ideas.